streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -134,143 +134,6 @@ public TopologyTestDriverTest(final boolean eosEnabled) {
         }
     }
 
-    private final static class Record {
-        private final Object key;
-        private final Object value;
-        private final long timestamp;
-        private final long offset;
-        private final String topic;
-        private final Headers headers;
```

Comment:
Seems we loose offset, topic, and header using `MockProcessor` -- is this intended?